### PR TITLE
Financial Assistance

### PR DESCRIPTION
**Pull request by doismellburning**

### CODE REVIEW

**Comment by doismellburning** [comment]
I specifically didn't make the link live because the form isn't live yet, but happy to tweak that separately/in this

**Comment by AnnJBarr** [comment]
That's great thanks @doismellburning. I've switched the google form on to accepting responses so happy for the page and link to go live on merge.

File: src/financial-assistance.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+title: Financial Assistance
+---
+
+Our financial assistance programme exists to support attendees who might otherwise find it difficult to attend the conference.
+We have a budget set aside for this purpose.
+
+The deadline for applications is 17:00 (Europe/London) on Friday, 17th June 2022.
```
Comment (trallard) [inline src/financial-assistance.md:9]: Would suggest adding an are we meeting yet link to the time to make it easier for folks to track timezones

File: src/financial-assistance.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+For example, they feel might embarrassed to ask for help,
+or they doubt that they are the right kind of person,
+or they think that they don't deserve it,
+or that other people are more deserving.
+We hope that you won't feel like this, and we will be glad to receive your request for support.
+
+We welcome applications from all attendees, and strongly encourage you to apply if it would make it easier for you to attend.
+Many of your fellow attendees will be doing the same, including many speakers and even members of the conference organisation team.
+
+This year we are currently only able to offer financial assistance in the form of free tickets.
```
Comment (trallard) [inline src/financial-assistance.md:27]:
```suggestion
Please note that this year we will only be able to offer financial assistance in the form of free tickets.
```

**Review by trallard — APPROVED** [review]
LGMT - make a couple of minor suggestions but none should be a blocker for merging

**Pull request merged**